### ISSUE TITLE
Community Infestation

### ISSUE THREAD

**Issue by DaCowardlyKY73** [mgraves9803/HavocMC-Official#3]
Certain and several communities that are supposed to be walled off of Walkers are infested with Walker Spawns. Infested Communities marked as **Bold** are not safe when two or above players are present in the area, while those marked with _Italics_ are safe, though there are certain areas where Walkers could still breach through either from certain climbable or breach areas, or random spawn rates somehow finding itself within the walls. The communities are:

1. **Hilltop**
2. _Alexandria_
3. _Kingdom_
4. **Sanctuary**
5. _Woodbury_
6. **Prison**

I would also suggest lowering the spawn rates in the following Group areas:
1. Hershel's Farm
2. Whisperer's Camp
As they have been considered secure.

**Comment by XxRagingKillerX** [comment]
Hershel's Farm got over run in the end of s4 so it should be bold

**Comment by DaCowardlyKY73** [comment]
As was Woodbury in the TV Series after it had been abandoned and burnt down, yet there they stand. I propose lowering spawn rates, not only so that there will be more room for exploration and expansion opportunities in the purpose of PvP base making or PvE Roleplays, but so that it doesn't become too hectic when there's too many players in the area. Additionally, walkers tend to spawn on the main house's roof, which is comical and an unnatural way for walkers to be in.

**Comment by DaCowardlyKY73** [comment]
**Hopeful Suggestive Fixes for the following issues...**
Note: This has been sent to the HavocMC Discord in [#]suggestions. However, it was kept short due to Discord's text limit.

_**1.) The Walkers**_
Several players have noted that the new Walkers are challenging, more frightening and a new change of pace - quite literally. The spawn rate is also a lovely change, allowing for players to experience the horror of the universe's horde. However, when the spawn rate is combined with the supposed walking Walkers speed, it becomes hectic and stressful for both experienced and beginning players alike. 
Although the purpose of the genre is to follow the trope of Horror, and the revamping of the Walkers to be a way of balancing the game as well as make them more frightening, there are other ways in which Walkers could be fixed:
**First**, keep the spawn rate. Everybody loves being in that one scene in either the comics or the TV show where they are nearly overrun, or if they are making a last stand. Plus, who wouldn't want to see a threatening amount of Walkers around one's area that would force them to tread carefully. It is an amazing feature, and should definitely be kept. However, spawn points should be double checked especially for certain communities as mentioned in the first comment.
**Second**, change their speed. Again, it has been both a stress for experienced and beginning players alike. Mix that speed with connection issues and glitches such as getting stuck in certain blocks and half-doors, people would surely be disappointed by how unbelievable the death would be. As much as I want to call the glitchy soil a feature, it does not seem worthy anymore now that any armored or unarmored player could be trampled over by two measly walkers, or just _one_ Whisperer. Prior the update, many players have coped with the glitchy blocks, especially the glitchy soil from farms or the wilderness. They were able to escape death several times, with little or no health loss armored or unarmored. Now, one glitch could sling you back and you could be killed by the mentioned number of mobs and the mentioned amount of protection. Either bring it back to the past speed and keep the high spawn rate for hordes, or make their speed slower than the recent update, but faster prior the update if possible within Minecraft's reach.

_**2.) The Whisperers**_
Nine years into the TV Series timeline, the walkers barely phase anyone anymore unless it's a horde, or a man or woman disguised. The Whisperers are the next big threat, and I love their addition to the game; the shock that you get whenever you try to melee a walker, the adrenaline that rushes through you when you try your best to circle around them or climb higher places... It is all quite an amazing decision to add them in. However, with the recent update, many have grown to despise them as a feature - not as a character. With the higher spawn rate, it is understandable that The Whisperers would also be affected. However, running in an open field causes at least _12_ to _14_ Whisperers encircling the player, plus the same speed supposed-walking Walkers around the area aggro'd to the player. In enclosed areas, at least _4_ to _6_ appear, if not more beyond, below or above the building the player is in.
For me, I love the amount of Whisperers that appear in the area. However, a single Whisperer is enough to take an armored or unarmored player, which is what makes it more unbelievable since the update. Pile in the glitches and/or connectivity issues, it would surely be a much more disappointing death. The Whisperer's damages had no longer been the old 2-hit-kill unarmored, and many players both beginning and experienced alike seek to enjoy a game without the bulky Minecraft armors. Now, there's a chance for the Whisperers to 1-hit-kill unarmored players, simply by holding a machete by luck - or misfortune, for those who dies to them. 
The problem about this damage is that it is no longer consistent, and there is barely any time to signal the players of what they are holding given how the colors for the texture of the weapons the Whisperers hold are nearly the same. As much as things depend on the reaction time, sometimes reactions are delayed by the medium - the game itself, by either the player's own connection or the glitches that put players back where they apparently were according to a single block. The damage should be consistent: 2-hit-kill is enough a wrath and mercy for players of all experiences.

In summary, Walkers shall return to a slower pace yet keep the large number through the high spawn rates to act as support for the new environmental threat: the Whisperers, who shall be able to eliminate players within 2 hits unarmored, around 4 or so armored, like the Walkers do on unarmored instead.

Nevertheless, the update shows promise of a great challenge and better opportunities. We just hope that challenges does not simply overwhelm the enjoyment of the game, with all the other issues it already presents.
